<!-- Top-Level launch file for bundled launching /-->
<!--Simulation launcher for the HCAUV aircraft-->

<launch>
	<arg name="manualMode" default="false"/>
	<arg name="autoMode" default="false"/>
	<!-- Launch the simulator /-->
	<include file="$(find last_letter)/launch/sim_HCUAV.launch" >
		<arg name="manualMode" value="$(arg manualMode)"/>
		<arg name="autoMode" value="$(arg autoMode)"/>
	</include>

	<!-- ArduPlane SITL connection -->
	<group>
		<node pkg="last_letter" type="fdmUDPSend.py" name="fdmUDPSend" />
		<node pkg="last_letter" type="fdmUDPReceive.py" name="fdmUDPReceive" />
	</group>

	<!-- Launch the visualizers /-->
	<include file="$(find last_letter)/launch/visualization.launch" >
		<arg name="model" value="HCUAV" />
	</include>

</launch>
